noir-projects/aztec-nr/aztec/src/state_vars/storage.nr

```diff
@@ -5,3 +5,9 @@ trait Storage<T> where T: Serialize<N> + Deserialize<N> {
         self.storage_slot
     }
 }
+
+struct Storable<N> {
```

Comment:
Could you elaborate on why this exists, and add a comment in the code?

Reply:
Done!